core/broadleaf-framework/src/main/resources/config/bc/sql/load_admin_permissions.sql

```diff
@@ -311,7 +311,7 @@ INSERT INTO BLC_ADMIN_PERMISSION_XREF (ADMIN_PERMISSION_ID, CHILD_PERMISSION_ID)
 -- currency read
 INSERT INTO BLC_ADMIN_PERMISSION_XREF (ADMIN_PERMISSION_ID, CHILD_PERMISSION_ID) VALUES (-102, -34);
 -- Catalog read
-INSERT INTO BLC_ADMIN_PERMISSION_XREF (ADMIN_PERMISSION_ID, CHILD_PERMISSION_ID) VALUES (-101, -53);
+INSERT INTO BLC_ADMIN_PERMISSION_XREF (ADMIN_PERMISSION_ID, CHILD_PERMISSION_ID) VALUES (-101, -52);
```

Comment:
Maybe instead of `-101` here should be `-102`.

Reply:
Fixed the -101/-102 issue.